### PR TITLE
Updates to exoplanet example

### PR DESCRIPTION
**Pull request by Azaya89**
Updates to `exoplanet` example to close #189

### CODE REVIEW

**Review by maximlt — COMMENTED** [review]
I still need to check the dev site to see how it looks.

File: exoplanets/exoplanets.ipynb [hunk 1]
```diff
@@ -607,11 +607,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "explorer = pn.Row(pn.Column(widgets, overlay, pn.Row(x_axis, y_axis), bound_scatter))\n",
+    "pn.extension(sizing_mode='stretch_width')\n",
     "\n",
-    "dashboard = pn.template.BootstrapTemplate(title='Exoplanets Explorer')\n",
-    "dashboard.main.append('Filter, color, and size the displayed exoplanets using the available widgets, and explore trends in the scatterplot below.')\n",
+    "accordion = pn.Accordion(('Background', '''For the past 25+ years, NASA has used ground- and space-based methods to [identify exoplanets](https://exoplanets.nasa.gov/exep/about/missions-instruments) (planets outside of our solar system). In the past ten years in particular, campaigns like Kepler, K2, and TESS have produced an explosion of results. To date, approximately 4,400 exoplanets have been identified, and over 3,000 potential exoplanet candidates have been discovered. \n",
```
[the remark this answers is not in the document]
Comment (Azaya89) [reply exoplanets/exoplanets.ipynb]: Done.

**Comment by maximlt** [comment]
@Azaya89 I pushed some minor changes in https://github.com/holoviz-topics/examples/pull/463/commits/8f1679b8aa7a538d97a604c478203259bb5d63ab.

There was an issue with the background card being displayed twice in the app.

image

I think the indentation of the multi-line string was occupying too much horizontal space (even more obvious on mobile). This led me to notice break lines in a multi-line string were displayed by default by the Markdown pane (not sure this should be the default, see https://github.com/holoviz/panel/issues/7535). I changed the code to revert that behavior.

image

Can you please run the notebook and the app and check everything is okay? 🙏

**Comment by Azaya89** [comment]
LGTM!

image

**Pull request merged**